package.json

```diff
@@ -33,42 +43,48 @@
   "homepage": "https://github.com/apollostack/apollo-proxy#readme",
   "dependencies": {
     "lodash.uniq": "^4.3.0",
-    "node-uuid": "^1.4.7"
+    "node-uuid": "^1.4.7",
+    "graphql-typings": "git://github.com/DxCx/graphql-typings.git#tmp-pkg"
```

Comment:
I've had to do some small changes to the typings (Verified against graphql-js base code)
and submitted a PR to @nitintutlani typed-graphql repo (Which is what apollo-server is using)
so we have 2 options here:
1. Wait for the merge and the issue of a new package.
2. i can create npm package for those typings and maintain it, then we can use it for both graphql-tools and apollo-server.